projects/packages/forms/src/dashboard/inbox/util.js

```diff
@@ -39,6 +41,20 @@ export const formatFieldValue = fieldValue => {
 		return join( fieldValue, ', ' );
 	}
 
+	if ( fieldValue.startsWith( '{' ) ) {
```

Comment:
I this is how we're checking for json? I wonder if there's a better way. This seems like an odd and potentially fragile conditional.

Reply:
I see we're doing a similar check on line 788 of class-admin.php.